docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -33,17 +33,16 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **ORDER BY**
   - **LIKE**
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
+  The last two operators, **ORDER BY** and **LIKE**, are only supported with Elasticsearch as a Visibility store.
+
+- To search for a specific token, [set a custom Search Attribute of type `Text`](/app-dev-context/observability#custom-search-attributes) that has the value of the token.
+  To search for multiple tokens, separate them by spaces or punctuation.
+  In the following example for [`tctl-next`](/tctl-next/workflow/list), the Query runs after setting the custom Search Attribute to `type1`:
+
+  ```
+  tctl workflow list --query "mySearchAttribute = \"type1\""
```

Comment:
This example is irrelevant here. If you want to describe how search attributes of type `Text` works you better set it to some phrase first and then search for specific word. I also would like to avoid names like "mySearchAttribute" and values like "type1". Of course, it is random text but by making it more meaningful you make example better. This why I used `ProductId`.

Reply:
There are examples below. Going to remove this bit.